IFC4x3/Properties/n/NumberOfPhase_0ZOkcOH$14AAoaednCd9ms/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfPhase_0ZOkcOH14AAoaednCd9ms" Name="NumberOfPhase" UniqueId="2362e998-47f0-4428-aca4-a27c4c9c9c36" PrimaryDataType="IfcInteger" />
```

Comment:
Should this not be plural i.e. `NumberOfPhases`?

Reply:
5a0823b99d71f09e36cb3aa03ad2359787558895